index.html

```diff
@@ -82,7 +82,221 @@ <h4>let's connect</h4>
+            sign-ups required.
+          </p>
+          <ul class="project-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="project-btn" type="button">See Project</button>
+        </div>
+        <div class="card">
+          <figure>
+            <img
+              src="assets/images/background_image/second_portfolio.png"
+              alt="project image"
+            />
+          </figure>
+          <h2>Multi-Post Stories</h2>
+          <div class="author-lang">
+            <span>CANOPY</span>
+            <ul>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="project-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="project-btn" type="button">See Project</button>
+        </div>
+        <div class="card">
+          <figure>
+            <img
+              src="assets/images/background_image/third_portfolio.png"
+              alt="project image"
+            />
+          </figure>
+          <h2>Tonic</h2>
+          <div class="author-lang">
+            <span>CANOPY</span>
+            <ul>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="project-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="project-btn" type="button">See Project</button>
+        </div>
+        <div class="card">
+          <figure>
+            <img
+              src="assets/images/background_image/fourth_portfolio.png"
+              alt="project image"
+            />
+          </figure>
+          <h2>Multi-Post Stories</h2>
+          <div class="author-lang">
+            <span>CANOPY</span>
+            <ul>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="project-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="project-btn" type="button">See Project</button>
+        </div>
+      </section>
+      <!--end of projects section-->
+      <section class="about">
+        <div class="about-section limited">
+          <h1>About <br />Myself</h1>
```

Comment:
- [ ] Really good job on the project 👍 the `<h1>` tag can only be used once however, it can still remain a header tag but pick `<h2>` for example.

Reply:
Thanks for the feedback Mike